src/com/lftechnology/java/training/sanish/application/model/domain/Employee.java

```diff
@@ -0,0 +1,171 @@
+    private static final Logger LOGGER = Logger.getLogger(Employee.class.getName());
+    private int employeeId;
+    private int userId;
+    private String fullName;
+    private String address;
+    private String department;
+    private String role;
+    private String createdAt;
+    private String modifiedAt;
+
+    public int getEmployeeId() {
+        return employeeId;
+    }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public String getFullName() {
+        return fullName;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public String getDepartment() {
+        return department;
+    }
+
+    public String getRole() {
+        return role;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public String getModifiedAt() {
+        return modifiedAt;
+    }
+
+    public void setEmployeeId(int employeeId) {
+        this.employeeId = employeeId;
+    }
+
+    public void setUserId(int userId) {
+        this.userId = userId;
+    }
+
+    public void setFullName(String fullName) {
+        this.fullName = fullName;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public void setDepartment(String department) {
+        this.department = department;
+    }
+
+    public void setRole(String role) {
+        this.role = role;
+    }
+
+    public void setCreatedAt(String createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public void setModifiedAt(String modifiedAt) {
+        this.modifiedAt = modifiedAt;
+    }
+
+    @Override public String getTable() {
+        return "employees";
+    }
+
+    @Override public <V> V getAttribute(String key) {
+        try {
+            return (V) Employee.class.getDeclaredField(key).get(this);
+        } catch (NoSuchFieldException e) {
+            LOGGER.log(Level.WARNING, Constants.EXCEPTION_ERROR_MSG_LABEL + "{0}", new Object[] { e });
+        } catch (IllegalAccessException e) {
+            LOGGER.log(Level.WARNING, Constants.EXCEPTION_ERROR_MSG_LABEL + "{0}", new Object[] { e });
+        }
+
+        return null;
+    }
+
+    @Override public <V> void setAttribute(String key, V value) {
+        try {
+            Employee.class.getDeclaredField(key).set(this, value);
+        } catch (NoSuchFieldException e) {
+            LOGGER.log(Level.WARNING, Constants.EXCEPTION_ERROR_MSG_LABEL + "{0}", new Object[] { e });
+        } catch (IllegalAccessException e) {
+            LOGGER.log(Level.WARNING, Constants.EXCEPTION_ERROR_MSG_LABEL + "{0}", new Object[] { e });
+        }
+    }
+
+    @Override public boolean equals(Employee employeeModel) {
+        if (employeeModel == null || getClass() != employeeModel.getClass()) {
+            return false;
+        } else if (this == employeeModel) {
+            return true;
+        } else {
+            if (employeeId == employeeModel.employeeId || userId == employeeModel.userId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1126)